core/chain_indexer.go

```diff
@@ -46,6 +46,10 @@ type ChainIndexerBackend interface {
 
 	// Commit finalizes the section metadata and stores it into the database.
 	Commit() error
+
+	// PruneSections deletes relative chain data associated with specified section
+	// range.
+	PruneSections(uint64) error
```

Comment:
Could we name this simply `Prune`? Also please add a parameter name.

Reply:
It's also not clear from the documentation which side of the chain is pruned (I assume genesis -> argument). Perhaps expand the comment that: `Prune deletes the chain index older than the given threshold.` (and perhaps call the parameter `threshold`).